### PR TITLE
Add task query parameters

### PR DESCRIPTION
**Pull request by thePanz**

### CODE REVIEW

File: cf_test.go [hunk 1]
```diff
@@ -8,6 +8,7 @@ import (
 
 	"github.com/go-martini/martini"
 	"github.com/martini-contrib/render"
+	"net/url"
```
Comment (mweibel) [inline cf_test.go]: please move net/url import below `net/url/httptest`

File: tasks.go [hunk 2]
```diff
@@ -7,6 +7,7 @@ import (
 	"io"
 	"io/ioutil"
 	"time"
+	"net/url"
```
Comment (mweibel) [inline tasks.go]: please move that in alphabetic order (`net/url` before `time`)

Comment (mweibel) [inline tasks.go]: please move that in alphabetic order (`net/url` before `time`)

**Comment by mweibel** [comment]
Looks good to me. 
I work together with @thePanz and we were wondering whether we should rather break backwards compatibility and have just one `func ListTasks(query url.Values)` instead of `func ListTasks()` and `func ListTasksByQuery(query url.Values)` or not. We went for the latter option (separate funcs) but maybe you guys prefer having only one?

**Comment by lnguyen** [comment]
@thePanz i merged in #82 and caused some conflicts, can you fix this?

**Comment by thePanz** [comment]
@lnguyen PR updated
With @mweibel's help I also applied some GO coding styles fixes

**Pull request merged**